server/Dockerfile

```diff
@@ -9,17 +9,23 @@ USER root
 
 RUN yum install -y epel-release && yum install -y jq && yum clean all
 
-USER jboss
-
 RUN cd /opt/jboss/ && curl -L https://downloads.jboss.org/keycloak/$KEYCLOAK_VERSION/keycloak-$KEYCLOAK_VERSION.tar.gz | tar zx && mv /opt/jboss/keycloak-$KEYCLOAK_VERSION /opt/jboss/keycloak
 
 ADD docker-entrypoint.sh /opt/jboss/
 
 ADD setLogLevel.xsl /opt/jboss/keycloak/
 RUN java -jar /usr/share/java/saxon.jar -s:/opt/jboss/keycloak/standalone/configuration/standalone.xml -xsl:/opt/jboss/keycloak/setLogLevel.xsl -o:/opt/jboss/keycloak/standalone/configuration/standalone.xml
+RUN sed -i 's/\(logger.handlers=CONSOLE\),FILE/\1/' keycloak/standalone/configuration/logging.properties
```

Comment:
Why do you want to log to a file when in a container? stdout/stderr streams are captured by the execution engine (i.e. Docker). You're right this has nothing to do with OpenShift, so don't mind removing it if it at least allows a community image to run on OpenShift, but while I was updating this file thought I'd tidy up a few other things too.